app/controllers/student_teams_controller.rb

```diff
@@ -32,7 +32,9 @@ def action_allowed?
     when 'create'
       current_user_has_id? student.user_id
     when 'edit', 'update'
-      current_user_has_id? team.user_id
+      if @team!=nil
```

Comment:
This should not be the case, but I'll check on it and see how I can resolve this